### PR TITLE
Feature9  file show file and hide file added in the code

### PR DESCRIPTION
**Pull request by amolkedar1996**
I Have raised Pr Show file and hide file

### CODE REVIEW

File: Feature2/hide.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>hide file</title>
```
Comment (ArunaKed) [inline Feature2/hide.html:6]: hide file change title

File: Feature2/show.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Show File</title>
```
Comment (ArunaKed) [inline Feature2/show.html:6]: show file add title

File: Feature2/show.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+<body>
+    <h1>This Will Show The Holder.</h1>
```
Comment (ArunaKed) [inline Feature2/show.html:9]: add lorem text after h1 tag

**Review by ArunaKed — CHANGES_REQUESTED** [review]

File: Feature2/hide.html (hunk 1 above)
Comment (amolkedar1996) [reply Feature2/hide.html]: hide file title added

File: Feature2/show.html (hunk 2 above)
Comment (amolkedar1996) [reply Feature2/show.html]: show file title added

File: Feature2/show.html (hunk 3 above)
Comment (amolkedar1996) [reply Feature2/show.html:9]: lorem text added after h1 tag

**Review by ArunaKed — APPROVED** [review]
Ok Will all done good job amol

**Pull request merged**